frontend/src/Components/ReviewsModal/ReviewsModal.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import "./ReviewsModal.css";
+
+const ReviewsModal = ({ businessId, onClose }) => {
+  const [reviews, setReviews] = useState([]);
+  const defaultProfilePhoto = "frontend/src/assets/image.png";
+
+  useEffect(() => {
+    const fetchReviews = async () => {
+      const response = await fetch(
+        `http://localhost:3000/api/reviews/${businessId}`
```

Comment:
does `/api/reviews/${businessId}` also work here?

Reply:
Don't think it would work because the dev environment is being run in two different ports, I believe. The frontend and backend run in separate dev servers.

Reply:
I configured the proxy in vite.config.js as follows and updated the fetch call in the component:
server: {
  proxy: {
    '/api': {
      target: 'http://localhost:3000',
      changeOrigin: true,
      rewrite: (path) => path.replace(/^\/api/, ''),
    },
  },
},

Despite this, I'm still encountering a 404 (Not Found) error. The request seems to be sent to http://localhost:5173 instead of being proxied to http://localhost:3000. Could you provide any additional guidance on this issue?
@sinap-fb @jose-esquivel

Reply:
Oh ok nvm! I thought it would Just Work.